script/install/rbenv.bash

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-ruby_version=2.3.8
+ruby_version=2.4.10
```

Comment:
Didn't see that this was still a draft, but my change shouldn't hurt.
I haven't tested my change yet.
(We should refactor these scripts some time so the version is only specified in one place -- should be pretty easy.)

Reply:
I dropped your commit out of existence by mistake, but i've re-implemented it and folded it into the main commit.